config/software/dep-selector-libgecode.rb

```diff
@@ -36,8 +38,11 @@
 
   # Ruby DevKit ships with BSD Tar
   env["PROG_TAR"] = "bsdtar" if windows?
+  env["ARFLAGS"] = "rv #{env["ARFLAGS"]}" if env["ARFLAGS"]
```

Comment:
What's really happening here is *omnibus* doesn't put `rv` into the `ARFLAGS` on purpose, because some projects expect it *not* to have the operation in the variable, and some projects expect it *to* have the operation there. There isn't consistency around this :(